docs/administrator-documentation/moderne-platform/references/reporting.md

```diff
@@ -39,7 +42,7 @@ These reports can be downloaded using the download button: ![](./assets/report-d
 All of these reports can be accessed via the API as well. Here is an example of how to prepare and download the recipe run report:
 
 ```graphql
-mutation firstGenerateRecipeRunReport {
+mutation firstDownloadRecipeRunReport {
```

Comment:
We wanted to change `firstGenerateUsageReport` to `firstDownloadUsageReport`, I'm porting this change to the previous documentation

Reply:
what does the `first` mean here?

Reply:
It is to help clarify that this mutation must be run first, below is the `secondDownloadRecipeRunReport` query